docs/source/common/lightning_module.rst

```diff
@@ -1474,18 +1470,6 @@ on_train_batch_end
 .. automethod:: pytorch_lightning.core.lightning.LightningModule.on_train_batch_end
     :noindex:
 
-on_epoch_start
-~~~~~~~~~~~~~~
-
-.. automethod:: pytorch_lightning.core.lightning.LightningModule.on_epoch_start
-    :noindex:
```

Comment:
lest write what shall be used instead and that it is deprecated?

Reply:
removing this on latest means we don't recommend it and thus reduce it's visibility since here  it's explicitly mentioned.